web/settings/modulconfigevu.php

```diff
@@ -1073,6 +1091,61 @@ function display_wattbezugmodul() {
 					$('#wizzarddoneForm').submit();
 				});
 
+				// load meter data from Fronius inverter
+				$('#wattbezugfroniusload').on("click",function() {
+					$('#wattbezugfroniusload').attr("disabled", true);
+					$('#wattbezugfroniusloadmessage').text("Lade Daten...");
+					$.getJSON('settings/froniusloadmeterdata.php?ip=' + $('#wrfroniusip').val(), function(data) {
```

Comment:
Gibt es einen Grund, warum Du ein lokales PHP Skript aufrufst und nicht direkt hier den WR abfragst?
Wenn es nur um ein Timeout-Handling geht, sollte anstatt `$.getJSON()` besser `$.ajax()` genutzt werden.

Reply:
Der Grund ist die [Same-Origin-Policy](https://de.wikipedia.org/wiki/Same-Origin-Policy). Das Skript wird in der Browser-Umgebung ausgeführt und darf daher keine Daten von einem anderem Host (hier Fronius WR) lesen. Durch das PHP Skript wird diese Abfrage serverseitig durchgeführt.